src/xpk/commands/info.py

```diff
@@ -0,0 +1,231 @@
+"""
+Copyright 2024 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+from ..utils import xpk_exit, xpk_print
+from ..core.kueue import prepare_kueuectl
+from .cluster import set_cluster_command
+from ..core.commands import (
+    run_command_for_value,
+)
+from ..core.core import (
+    add_zone_and_project,
+)
+import json
+from tabulate import tabulate
+from argparse import Namespace
+
+table_fmt = 'plain'
+
+
+def info(args: Namespace) -> None:
+  """Provide info about localqueues, clusterqueues and their resources.
```

Comment:
Localqueue, Clusterqueue, Flavour, ...  are kueue concepts. Is telling the user about these concepts in line with the Experience we are working on now? or are we going to translate them to something familiar to people coming from the other worlds?

This comment is probably concerns PMs. I ask because we are installing/configuring kueue on the cluster in the background and I'm wonder if we should assume users already have the knowledge about kueue.

Reply:
Probably shoud be named partition or sth, but I thing I will leave it as it is now, and we can come back to this topic after consultation.

Reply:
I would bring it up in the team's chat before completing this PR and get the go-ahead.